### PR TITLE
Add setup for getting started guide

### PR DESCRIPTION
**Pull request by fmisir**
This PR introduces a docker-compose file based on pre build images.
The images are built currently by me locally:

EDC images are based on EDC milestone-3.1.
API Wrapper / Proxy / Backend Service is based on the "merge-aas-setup" branch. 
Digital Twin Registry is the latest version from the INT environment.

As @florianrusch-zf  suggested, i pushed the images to `ghcr.io/catenax-ng/catenax-at-home/*` they are publicly available.

I had build issues because the gradle-wrapper.jar were not available, I added them as well.

Please don't merge this yet, as it's still work in progress.

### CODE REVIEW

**Comment by florianrusch-zf** [comment]
> [...] I had build issues because the gradle-wrapper.jar were not available, I added them as well.

if you use the `gradlew.sh|.bat` file it should download the jar file...or I'm wrong?

**Comment by fmisir** [comment]
> > [...] I had build issues because the gradle-wrapper.jar were not available, I added them as well.
> 
> if you use the `gradlew.sh|.bat` file it should download the jar file...or I'm wrong?

The gradle-wrapper.jar is used by the gradlew.sh to download gradle. So it's required to be there.

**Comment by florianrusch-zf** [comment]
> The gradle-wrapper.jar is used by the gradlew.sh to download gradle. So it's required to be there.

Ahhh shit, okay...again learned something! Thanks!

File: getting-started-guide/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  registry-service:
+    image: ghcr.io/catenax-ng/catenax-at-home/registry:0.0.1
+    environment:
+      SPRING_PROFILES_ACTIVE: local
+      IDP_ISSUER_URI: ""
+      SPRING_DATASOURCE_URL: "jdbc:h2:file:./semantics/persistence;CASE_INSENSITIVE_IDENTIFIERS=TRUE"
+    ports:
+      - "4243:4243" # default
+      - "5013:8090"
```
Comment (florianrusch-zf) [inline getting-started-guide/docker-compose.yml:87]: Missing EOL
```suggestion
      - "5013:8090"

```

File: .gitignore [hunk 2]
```diff
@@ -18,7 +18,8 @@
 *.zip
 *.tar.gz
 *.rar
-
+# gradle wrapper not to be ignored
+!**/gradle/wrapper/gradle-wrapper.jar
```
Comment (florianrusch-zf) [inline .gitignore:22]:
```suggestion
# maven & gradle wrapper not to be ignored
!**/.mvn/wrapper/maven-wrapper.jar
!**/gradle/wrapper/gradle-wrapper.jar

```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

**Comment by florianrusch-zf** [comment]
@fmisir can you then also add the `maven-wrapper.jar` files? I also suggest a change of the `.gitignore`

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**